common/components/common/projects/ProfileProjectSearch.jsx

```diff
@@ -11,16 +11,29 @@ import CollapsiblePreviewPanel from "../CollapsiblePreviewPanel.jsx";
 
 type Props = {|
   viewOnly: boolean,
-  wide: boolean,
+  wide: boolean
 |};
 
-class ProfileProjectSearch extends React.PureComponent<Props> {
+type State = {|
+  hidden: boolean
```

Comment:
Let's either reverse the logic or pick a different name for this, as hidden gets set to false when we want to hide the section.